From 4fb61d7c621599b0db9c2c45f4243f07936c5953 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 13 Oct 2021 08:20:00 -0700
Subject: [PATCH] setup.py: Do not use 2to3

This helps compiling with setuptools 0.58+
and it seems to compile fine with python3

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 setup.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/setup.py b/setup.py
index 9fe4902..f2098e1 100644
--- a/setup.py
+++ b/setup.py
@@ -2,8 +2,6 @@ import os
 import sys
 
 extra = {}
-if sys.version_info >= (3, 0):
-    extra.update(use_2to3=True)
 
 try:
     from setuptools import setup, find_packages
-- 
2.33.0

